app/models/manageiq/providers/redhat/infra_manager/provision/configuration.rb

```diff
@@ -13,6 +13,10 @@ def attach_floppy_payload
   def configure_cloud_init
     return unless content = customization_template_content
     get_provider_destination.cloud_init = content
+
+    if Gem::Version.new(source.ext_management_system.api_version) >= Gem::Version.new("3.5.5.0")
```

Comment:
We only want to set this if we're talking to RHEV 3.5.5.0 or above, I used Gem::Version to compare the versions.